gnome-extra/cinnamon/cinnamon-4.8.5.ebuild

```diff
@@ -0,0 +1,173 @@
+	)
+	networkmanager? (
+		net-misc/networkmanager[introspection]
+	)
+"
+# caribou used by onscreen keyboard
+# libtimezonemap used by datetime settings
+# iso-flag-png (unpackaged) used by keyboard layout settings
+RDEPEND="${COMMON_DEPEND}
+	>=app-accessibility/caribou-0.3
+	dev-libs/keybinder:3[introspection]
+	dev-libs/libtimezonemap
+	$(python_gen_cond_dep '
+		dev-python/dbus-python[${PYTHON_USEDEP}]
+		dev-python/distro[${PYTHON_USEDEP}]
+		dev-python/pexpect[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+		dev-python/pyinotify[${PYTHON_USEDEP}]
+		dev-python/pypam[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+		dev-python/setproctitle[${PYTHON_USEDEP}]
+		dev-python/tinycss2[${PYTHON_USEDEP}]
+		dev-python/xapp[${PYTHON_USEDEP}]
+	')
+	>=gnome-base/dconf-0.4.1
+	>=gnome-base/gsettings-desktop-schemas-2.91.91
+	>=gnome-base/libgnomekbd-2.91.4
+	>=gnome-extra/cinnamon-control-center-4.8[networkmanager=]
+	>=gnome-extra/cinnamon-screensaver-4.8
+	>=gnome-extra/cinnamon-session-4.8
+	>=gnome-extra/cinnamon-settings-daemon-4.8
+	>=gnome-extra/nemo-4.8
+	gnome-extra/polkit-gnome
+	net-misc/wget
+	sys-apps/accountsservice[introspection]
+	sys-power/upower[introspection]
+	>=x11-libs/xapps-2.0.5[introspection]
+	x11-misc/xdg-utils
+	x11-themes/adwaita-icon-theme
+	x11-themes/gnome-themes-standard
+
+	nls? (
+		>=gnome-extra/cinnamon-translations-4.8
+	)
+"
+DEPEND="${COMMON_DEPEND}
+	>=dev-util/intltool-0.40
+	>=sys-devel/gettext-0.17
+	virtual/pkgconfig
+
+	gtk-doc? ( dev-util/gtk-doc )
+"
+
+PATCHES=(
+	# Fix backgrounds path as cinnamon doesn't provide them
+	# https://github.com/linuxmint/Cinnamon/issues/3575
+	"${FILESDIR}"/${PN}-3.8.0-gnome-background-compatibility.patch
+
+	# Use wheel group instead of sudo (from Fedora/Arch)
+	# https://github.com/linuxmint/Cinnamon/issues/3576
+	"${FILESDIR}"/${PN}-3.6.6-wheel-sudo.patch
+
+	# GStreamer only needed for recorder
+	# https://github.com/linuxmint/cinnamon/pull/9809
+	"${FILESDIR}"/${PN}-4.8.5-build-fixes.patch
+)
+
+src_prepare() {
+	xdg_src_prepare
+
+	# Add polkit agent to required components
+	# https://github.com/linuxmint/Cinnamon/issues/3579
+	sed -i "s/'REQUIRED', '/&polkit-cinnamon-authentication-agent-1;/" meson.build || die
+
+	# shebang fixing craziness
+	local p
+	for p in $(grep -rl '#!.*python3'); do
+		python_fix_shebang "${p}"
+	done
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use gstreamer build_recorder)
+		$(meson_use gtk-doc docs)
+		-Ddisable_networkmanager=$(usex networkmanager false true)
+	)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+
+	python_optimize "${ED}"/usr/share/cinnamon/
+
+	# Required for gnome-shell on hardened/PaX, bug #398941
+	pax-mark mr "${ED}"/usr/bin/cinnamon
+
+	# Doesn't exist on Gentoo, causing this to be a dead symlink
+	rm -f "${ED}/etc/xdg/menus/cinnamon-applications-merged" || die
```

Comment:
We avoid using `rm -f` because it hides errorneous calls and then they're just left in the ebuild for an eternity. So remove the `-f`, it's only used if something can't be predicted.